### ISSUE TITLE
dynmapblockscan crash

### ISSUE THREAD

**Issue by esotericist** [webbukkit/DynmapBlockScan#2]
Decided to give this a whirl, since I'd like to see dynmap on my lost cities server in the hypothetical future...

My work-in-progress server pack has.. quite a lot of stuff in it, so I half expected this to fail. I suppose in that respect I wasn't disappointed?

Here's the full console log, since there wasn't a proper crash report:
https://gist.github.com/esotericist/d2d79362c317282d632c3ed42746770e

Being heavily modded minecraft, there's a LOT of noise here, and in particular there's a lot of journeymap stuff at the end that isn't pertinent. I'm pretty sure for your purposes, the stuff you want is at [18:10:22].

**Comment by esotericist** [comment]
Notably, the error looks like there might be some broken json in one of the mods, but there isn't any indication of what file was being scanned at the time, so I can't investigate that end.

**Comment by mikeprimm** [comment]
OK - I've updated the DynmapBlockScan code to help sort this out: it should prevent the hard-crash, but also should be logging which file in which mod has the 'malformed JSON'.  Please share what you see!

**Comment by esotericist** [comment]
Thanks so much for getting back on this. 

I won't be in a position to try this for at least another 9 or 10 hours, but I'll definitely report back once I do.

**Comment by esotericist** [comment]
well, the good news is, it didn't crash, so my original problem has been addressed.
The bad news is, it made a LOT of noise in the log.

I broke it up into two seconds. The log for loading the game, then the log for loading the world.
https://gist.github.com/esotericist/4e5690f59e3d30b431ca03554c04772b

The first bit is truncated, obviously. and the second one, being almost 24k lines, doesn't even show a preview in gist.
